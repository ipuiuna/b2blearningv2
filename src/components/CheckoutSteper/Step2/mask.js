export const cepMask = (value) => {
  if (!value) return value;
  value = value.replace(/\D/g, '');
  value = value.replace(/(\d{5})(\d)/, '$1-$2');
  value = value.replace(/(-d{3})\d+?$/, '$1');
  return value;
};

export const numberMask = (value) => {
  if (!value) return value;
  return value.replace(/\D/g, '');
};
