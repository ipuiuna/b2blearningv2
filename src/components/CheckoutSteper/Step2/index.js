import React from 'react';
import { TextField, Grid, Box, FormControl } from '@material-ui/core';
import useStyles from './styles';
import { cepMask, numberMask } from './mask';

const TextControl = (props) => {
  const classes = useStyles();
  return (
    <FormControl fullWidth>
      <TextField className={classes.textField} color='primary' {...props} />
    </FormControl>
  );
};

export default function Step2(props) {
  const { setFormState, formState } = props;

  const handleChangeEvt = (evt) => {
    const fieldName = evt.target.name;
    if (evt.target.name !== 'cep') {
      setFormState({ ...formState, [fieldName]: evt.target.value });
    } else {
      if (evt.target.value.length > 9) {
        setFormState({ ...formState, [fieldName]: formState.cep });
      } else {
        setFormState({ ...formState, [fieldName]: evt.target.value });
      }
    }
  };

  return (
    <Box
      boxShadow={2}
      style={{ padding: '0 10%', backgroundColor: 'rgba(229, 229, 229, 0.35)' }}
    >
      <Grid
        direction='column'
        container
        style={{
          paddingBottom: '50px',
        }}
      >
        <Grid container direction='row'>
          <Grid item xs={12}>
            <TextControl
              name='nome'
              id='name'
              helperText={
                formState.nome && formState.nome.length > 128
                  ? 'Numero de caracteres máximo atingido.'
                  : formState.nome === ''
                  ? 'Por favor preencha seu nome'
                  : ''
              }
              label='Nome completo'
              onChange={handleChangeEvt}
            />
          </Grid>
          <Grid item sm={6} xs={12}>
            <TextControl
              label='Rua'
              helperText={
                formState.rua === '' ? 'Digite sua rua por favor.' : ''
              }
              id='address'
              name='rua'
              onChange={handleChangeEvt}
            />
          </Grid>
          <Grid item sm={2} xs={12}>
            <TextControl
              label='Número'
              helperText={
                formState.numero === '' ? 'Digite seu número por favor.' : ''
              }
              id='number'
              placeholder='123'
              name='numero'
              value={numberMask(formState.numero)}
              onChange={handleChangeEvt}
            />
          </Grid>
          <Grid item sm={4} xs={12}>
            <TextControl
              label='Bairro'
              helperText={
                formState.bairro === '' ? 'Digite seu bairro por favor.' : ''
              }
              name='bairro'
              id='bairro'
              onChange={handleChangeEvt}
            />
          </Grid>
          <Grid item sm={6} xs={12}>
            <TextControl
              label='Cidade'
              helperText={
                formState.cidade === '' ? 'Digite sua cidade por favor.' : ''
              }
              name='cidade'
              id='city'
              onChange={handleChangeEvt}
            />
          </Grid>
          <Grid item sm={3} xs={12}>
            <TextControl
              label='Estado'
              name='estado'
              helperText={
                formState.estado === '' ? 'Digite seu estado por favor.' : ''
              }
              id='state'
              onChange={handleChangeEvt}
            />
          </Grid>
          <Grid item sm={3} xs={12}>
            <TextControl
              label='Cep'
              name='cep'
              id='cep'
              placeholder='00000-000'
              value={cepMask(formState.cep)}
              helperText={
                (formState.cep && formState.cep.length !== 9) ||
                formState.cep === ''
                  ? 'Por favor digite um CEP válido.'
                  : ''
              }
              onChange={handleChangeEvt}
            />
          </Grid>
        </Grid>
      </Grid>
    </Box>
  );
}
