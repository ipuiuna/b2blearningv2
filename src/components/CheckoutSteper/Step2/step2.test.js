import React from 'react';
import renderer from 'react-test-renderer';

import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import Step2 from './';

import { TextField } from '@material-ui/core';

configure({ adapter: new Adapter() });

const formState = {
  nome: '',
  rua: '',
  cep: '',
  numero: '',
  estado: '',
  cidade: '',
  bairro: '',
};

describe('<Step2 />', () => {
  let wrapper;
  it('renders correctly', () => {
    const tree = renderer.create(<Step2 formState={formState} />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  //   beforeEach(() => {
  //     wrapper = shallow(
  //       <Step2
  //         total={0}
  //         getCart={() => {
  //           return 0;
  //         }}
  //       />
  //     );
  //   });

  //   it('Should have a field to enter "nome"', () => {
  //     expect(wrapper.find('input').prop('id')).toEqual('name');
  //   });

  //   it('Should have a field to enter "rua"', () => {
  //     expect(wrapper.find(Cart)).toHaveLength(1);
  //   });

  //   it('Should have a field to enter "numero"', () => {
  //     expect(wrapper.find(Cart)).toHaveLength(1);
  //   });

  //   it('Should have a field to enter "bairro"', () => {
  //     expect(wrapper.find(Cart)).toHaveLength(1);
  //   });

  //   it('Should have a field to enter "cidade"', () => {
  //     expect(wrapper.find(Cart)).toHaveLength(1);
  //   });

  //   it('Should have a field to enter "estado"', () => {
  //     expect(wrapper.find(Cart)).toHaveLength(1);
  //   });

  //   it('Should have a field to enter "cep"', () => {
  //     expect(wrapper.find(Cart)).toHaveLength(1);
  //   });
});
