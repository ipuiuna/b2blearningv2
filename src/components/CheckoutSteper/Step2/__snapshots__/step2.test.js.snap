// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<div
  className="MuiBox-root MuiBox-root-2"
  style={
    Object {
      "backgroundColor": "rgba(229, 229, 229, 0.35)",
      "padding": "0 10%",
    }
  }
>
  <div
    className="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-column"
    style={
      Object {
        "paddingBottom": "50px",
      }
    }
  >
    <div
      className="MuiGrid-root MuiGrid-container"
    >
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
      >
        <div
          className="MuiFormControl-root MuiFormControl-fullWidth"
        >
          <div
            className="MuiFormControl-root MuiTextField-root makeStyles-textField-106"
          >
            <label
              className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
              data-shrink={false}
              htmlFor="name"
              id="name-label"
            >
              Nome completo
            </label>
            <div
              className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
              onClick={[Function]}
            >
              <input
                aria-describedby="name-helper-text"
                aria-invalid={false}
                autoFocus={false}
                className="MuiInputBase-input MuiInput-input"
                disabled={false}
                id="name"
                name="nome"
                onAnimationStart={[Function]}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                required={false}
                type="text"
              />
            </div>
            <p
              className="MuiFormHelperText-root"
              id="name-helper-text"
            >
              Por favor preencha seu nome
            </p>
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6"
      >
        <div
          className="MuiFormControl-root MuiFormControl-fullWidth"
        >
          <div
            className="MuiFormControl-root MuiTextField-root makeStyles-textField-106"
          >
            <label
              className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
              data-shrink={false}
              htmlFor="address"
              id="address-label"
            >
              Rua
            </label>
            <div
              className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
              onClick={[Function]}
            >
              <input
                aria-describedby="address-helper-text"
                aria-invalid={false}
                autoFocus={false}
                className="MuiInputBase-input MuiInput-input"
                disabled={false}
                id="address"
                name="rua"
                onAnimationStart={[Function]}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                required={false}
                type="text"
              />
            </div>
            <p
              className="MuiFormHelperText-root"
              id="address-helper-text"
            >
              Digite sua rua por favor.
            </p>
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-2"
      >
        <div
          className="MuiFormControl-root MuiFormControl-fullWidth"
        >
          <div
            className="MuiFormControl-root MuiTextField-root makeStyles-textField-106"
          >
            <label
              className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
              data-shrink={false}
              htmlFor="number"
              id="number-label"
            >
              Número
            </label>
            <div
              className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
              onClick={[Function]}
            >
              <input
                aria-describedby="number-helper-text"
                aria-invalid={false}
                autoFocus={false}
                className="MuiInputBase-input MuiInput-input"
                disabled={false}
                id="number"
                name="numero"
                onAnimationStart={[Function]}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                placeholder="123"
                required={false}
                type="text"
                value=""
              />
            </div>
            <p
              className="MuiFormHelperText-root"
              id="number-helper-text"
            >
              Digite seu número por favor.
            </p>
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-4"
      >
        <div
          className="MuiFormControl-root MuiFormControl-fullWidth"
        >
          <div
            className="MuiFormControl-root MuiTextField-root makeStyles-textField-106"
          >
            <label
              className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
              data-shrink={false}
              htmlFor="bairro"
              id="bairro-label"
            >
              Bairro
            </label>
            <div
              className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
              onClick={[Function]}
            >
              <input
                aria-describedby="bairro-helper-text"
                aria-invalid={false}
                autoFocus={false}
                className="MuiInputBase-input MuiInput-input"
                disabled={false}
                id="bairro"
                name="bairro"
                onAnimationStart={[Function]}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                required={false}
                type="text"
              />
            </div>
            <p
              className="MuiFormHelperText-root"
              id="bairro-helper-text"
            >
              Digite seu bairro por favor.
            </p>
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6"
      >
        <div
          className="MuiFormControl-root MuiFormControl-fullWidth"
        >
          <div
            className="MuiFormControl-root MuiTextField-root makeStyles-textField-106"
          >
            <label
              className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
              data-shrink={false}
              htmlFor="city"
              id="city-label"
            >
              Cidade
            </label>
            <div
              className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
              onClick={[Function]}
            >
              <input
                aria-describedby="city-helper-text"
                aria-invalid={false}
                autoFocus={false}
                className="MuiInputBase-input MuiInput-input"
                disabled={false}
                id="city"
                name="cidade"
                onAnimationStart={[Function]}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                required={false}
                type="text"
              />
            </div>
            <p
              className="MuiFormHelperText-root"
              id="city-helper-text"
            >
              Digite sua cidade por favor.
            </p>
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-3"
      >
        <div
          className="MuiFormControl-root MuiFormControl-fullWidth"
        >
          <div
            className="MuiFormControl-root MuiTextField-root makeStyles-textField-106"
          >
            <label
              className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
              data-shrink={false}
              htmlFor="state"
              id="state-label"
            >
              Estado
            </label>
            <div
              className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
              onClick={[Function]}
            >
              <input
                aria-describedby="state-helper-text"
                aria-invalid={false}
                autoFocus={false}
                className="MuiInputBase-input MuiInput-input"
                disabled={false}
                id="state"
                name="estado"
                onAnimationStart={[Function]}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                required={false}
                type="text"
              />
            </div>
            <p
              className="MuiFormHelperText-root"
              id="state-helper-text"
            >
              Digite seu estado por favor.
            </p>
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-3"
      >
        <div
          className="MuiFormControl-root MuiFormControl-fullWidth"
        >
          <div
            className="MuiFormControl-root MuiTextField-root makeStyles-textField-106"
          >
            <label
              className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
              data-shrink={false}
              htmlFor="cep"
              id="cep-label"
            >
              Cep
            </label>
            <div
              className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
              onClick={[Function]}
            >
              <input
                aria-describedby="cep-helper-text"
                aria-invalid={false}
                autoFocus={false}
                className="MuiInputBase-input MuiInput-input"
                disabled={false}
                id="cep"
                name="cep"
                onAnimationStart={[Function]}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                placeholder="00000-000"
                required={false}
                type="text"
                value=""
              />
            </div>
            <p
              className="MuiFormHelperText-root"
              id="cep-helper-text"
            >
              Por favor digite um CEP válido.
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`<Step2 /> renders correctly 1`] = `
<div
  className="MuiBox-root MuiBox-root-2"
  style={
    Object {
      "backgroundColor": "rgba(229, 229, 229, 0.35)",
      "padding": "0 10%",
    }
  }
>
  <div
    className="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-column"
    style={
      Object {
        "paddingBottom": "50px",
      }
    }
  >
    <div
      className="MuiGrid-root MuiGrid-container"
    >
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
      >
        <div
          className="MuiFormControl-root MuiFormControl-fullWidth"
        >
          <div
            className="MuiFormControl-root MuiTextField-root makeStyles-textField-106"
          >
            <label
              className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
              data-shrink={false}
              htmlFor="name"
              id="name-label"
            >
              Nome completo
            </label>
            <div
              className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
              onClick={[Function]}
            >
              <input
                aria-describedby="name-helper-text"
                aria-invalid={false}
                autoFocus={false}
                className="MuiInputBase-input MuiInput-input"
                disabled={false}
                id="name"
                name="nome"
                onAnimationStart={[Function]}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                required={false}
                type="text"
              />
            </div>
            <p
              className="MuiFormHelperText-root"
              id="name-helper-text"
            >
              Por favor preencha seu nome
            </p>
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6"
      >
        <div
          className="MuiFormControl-root MuiFormControl-fullWidth"
        >
          <div
            className="MuiFormControl-root MuiTextField-root makeStyles-textField-106"
          >
            <label
              className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
              data-shrink={false}
              htmlFor="address"
              id="address-label"
            >
              Rua
            </label>
            <div
              className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
              onClick={[Function]}
            >
              <input
                aria-describedby="address-helper-text"
                aria-invalid={false}
                autoFocus={false}
                className="MuiInputBase-input MuiInput-input"
                disabled={false}
                id="address"
                name="rua"
                onAnimationStart={[Function]}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                required={false}
                type="text"
              />
            </div>
            <p
              className="MuiFormHelperText-root"
              id="address-helper-text"
            >
              Digite sua rua por favor.
            </p>
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-2"
      >
        <div
          className="MuiFormControl-root MuiFormControl-fullWidth"
        >
          <div
            className="MuiFormControl-root MuiTextField-root makeStyles-textField-106"
          >
            <label
              className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
              data-shrink={false}
              htmlFor="number"
              id="number-label"
            >
              Número
            </label>
            <div
              className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
              onClick={[Function]}
            >
              <input
                aria-describedby="number-helper-text"
                aria-invalid={false}
                autoFocus={false}
                className="MuiInputBase-input MuiInput-input"
                disabled={false}
                id="number"
                name="numero"
                onAnimationStart={[Function]}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                placeholder="123"
                required={false}
                type="text"
                value=""
              />
            </div>
            <p
              className="MuiFormHelperText-root"
              id="number-helper-text"
            >
              Digite seu número por favor.
            </p>
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-4"
      >
        <div
          className="MuiFormControl-root MuiFormControl-fullWidth"
        >
          <div
            className="MuiFormControl-root MuiTextField-root makeStyles-textField-106"
          >
            <label
              className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
              data-shrink={false}
              htmlFor="bairro"
              id="bairro-label"
            >
              Bairro
            </label>
            <div
              className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
              onClick={[Function]}
            >
              <input
                aria-describedby="bairro-helper-text"
                aria-invalid={false}
                autoFocus={false}
                className="MuiInputBase-input MuiInput-input"
                disabled={false}
                id="bairro"
                name="bairro"
                onAnimationStart={[Function]}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                required={false}
                type="text"
              />
            </div>
            <p
              className="MuiFormHelperText-root"
              id="bairro-helper-text"
            >
              Digite seu bairro por favor.
            </p>
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6"
      >
        <div
          className="MuiFormControl-root MuiFormControl-fullWidth"
        >
          <div
            className="MuiFormControl-root MuiTextField-root makeStyles-textField-106"
          >
            <label
              className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
              data-shrink={false}
              htmlFor="city"
              id="city-label"
            >
              Cidade
            </label>
            <div
              className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
              onClick={[Function]}
            >
              <input
                aria-describedby="city-helper-text"
                aria-invalid={false}
                autoFocus={false}
                className="MuiInputBase-input MuiInput-input"
                disabled={false}
                id="city"
                name="cidade"
                onAnimationStart={[Function]}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                required={false}
                type="text"
              />
            </div>
            <p
              className="MuiFormHelperText-root"
              id="city-helper-text"
            >
              Digite sua cidade por favor.
            </p>
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-3"
      >
        <div
          className="MuiFormControl-root MuiFormControl-fullWidth"
        >
          <div
            className="MuiFormControl-root MuiTextField-root makeStyles-textField-106"
          >
            <label
              className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
              data-shrink={false}
              htmlFor="state"
              id="state-label"
            >
              Estado
            </label>
            <div
              className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
              onClick={[Function]}
            >
              <input
                aria-describedby="state-helper-text"
                aria-invalid={false}
                autoFocus={false}
                className="MuiInputBase-input MuiInput-input"
                disabled={false}
                id="state"
                name="estado"
                onAnimationStart={[Function]}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                required={false}
                type="text"
              />
            </div>
            <p
              className="MuiFormHelperText-root"
              id="state-helper-text"
            >
              Digite seu estado por favor.
            </p>
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-3"
      >
        <div
          className="MuiFormControl-root MuiFormControl-fullWidth"
        >
          <div
            className="MuiFormControl-root MuiTextField-root makeStyles-textField-106"
          >
            <label
              className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
              data-shrink={false}
              htmlFor="cep"
              id="cep-label"
            >
              Cep
            </label>
            <div
              className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
              onClick={[Function]}
            >
              <input
                aria-describedby="cep-helper-text"
                aria-invalid={false}
                autoFocus={false}
                className="MuiInputBase-input MuiInput-input"
                disabled={false}
                id="cep"
                name="cep"
                onAnimationStart={[Function]}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                placeholder="00000-000"
                required={false}
                type="text"
                value=""
              />
            </div>
            <p
              className="MuiFormHelperText-root"
              id="cep-helper-text"
            >
              Por favor digite um CEP válido.
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
